.circleci/config.yml

```diff
@@ -25,3 +25,6 @@ jobs:
       - run:
           command: ./.circleci/run.sh betterc
           name: Run @betterC tests
+      - run:
+          command: ./.circleci/run.sh publictests
+          name: Run public unittests
```

Comment:
Just as an FYI this won't apply to all existing PRs. They need to be rebased for them to have this test.